rust/connlib/tunnel/src/control_protocol/client.rs

```diff
@@ -84,7 +84,7 @@ where
         relays: Vec<Relay>,
         reference: Option<Reference>,
     ) -> Result<Request> {
-        tracing::trace!("request_connection");
+        tracing::trace!(%gateway_id, %resource_id, "request_connection");
```

Comment:
the instrument already print these but in macos it's hard to see in the console, been quite useful to debug.